tests/subsys/mgmt/mcumgr/fs_mgmt_hash_supported/configuration/crc32.conf

```diff
@@ -4,3 +4,4 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 CONFIG_MCUMGR_GRP_FS_CHECKSUM_IEEE_CRC32=y
+CONFIG_MCUMGR_GRP_FS_HASH_SHA256=n
```

Comment:
this is not needed, not sure why it's here?

Reply:
This is out of consistency with `sha256.conf`.
If CRC32 is enabled (which it is by default) it will be set as the default (`MCUMGR_GRP_FS_CHECKSUM_HASH_DEFAULT` in `fs_mgmt.c`) so it felt like explicitly disabling it in the case of SHA-256 was a good idea.
Just doing the same thing the other way around here even though as of now it's not necessary.